DistanceToolColorizer.py

```diff
@@ -0,0 +1,35 @@
+# DistanceToolColorVisibilty
```

Comment:
If you have the intention to explain what your code does, be more explicit and concise